css/style.css

```diff
@@ -36,10 +36,27 @@ body {
   margin-left: 30px;
 }
 
+.my-mob-menu a {
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #edecff;
+}
+
 .my-nav-list {
   display: none;
 }
 
+/* .my-nav-list > li > a {
+  color: inherit;
+  text-decoration: inherit;
+} */
+
```

Comment:
[OPTIONAL]

- Your UI and code structure look really amazing 🤸‍♂️ , but things could be prettier without COMMENTED CODE 😢 
Kindly remove the commented lines of code because You Ain't Gonna Need It(YAGNI) 😉